Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Staging/SetPoints/Subsequences/EfficiencyCondition.mo

```diff
@@ -0,0 +1,293 @@
+  connect(div.y, hys2.u)
+    annotation (Line(points={{-58,20},{-42,20}},
+      color={0,0,127}));
+  connect(truDel1.u, hys1.y)
+    annotation (Line(points={{-2,80},{-18,80}},
+      color={255,0,255}));
+  connect(truDel.u, hys2.y)
+    annotation (Line(points={{-2,20},{-18,20}},
+      color={255,0,255}));
+  connect(intToRea.u, uTyp)
+    annotation (Line(points={{-102,-100},{-140,-100}},
+      color={255,127,0}));
+  connect(extIndSig.u, intToRea.y)
+    annotation (Line(points={{-62,-100},{-78,-100}},
+      color={0,0,127}));
+  connect(extIndSig.index, uAvaUp)
+    annotation (Line(points={{-50,-112},{-50,-140},{-140,-140}},
+      color={255,127,0}));
+  connect(greThr.u, extIndSig.y)
+    annotation (Line(points={{-22,-100},{-38,-100}},
+      color={0,0,127}));
+  connect(greThr.y, logSwi.u2)
+    annotation (Line(points={{2,-100},{30,-100},{30,50},{38,50}},
+      color={255,0,255}));
+  connect(truDel.y, logSwi.u3)
+    annotation (Line(points={{22,20},{36,20},{36,42},{38,42}},
+      color={255,0,255}));
+  connect(truDel1.y, logSwi.u1)
+    annotation (Line(points={{22,80},{36,80},{36,58},{38,58}},
+      color={255,0,255}));
+  connect(and2.y, yEffCon)
+    annotation (Line(points={{102,0},{140,0}},
+      color={255,0,255}));
+  connect(logSwi.y, and2.u1)
+    annotation (Line(points={{62,50},{70,50},{70,0},{78,0}},
+      color={255,0,255}));
+  connect(logSwi1.y, and2.u2)
+    annotation (Line(points={{62,-30},{70,-30},{70,-8},{78,-8}},
+      color={255,0,255}));
+  connect(con1.y, logSwi1.u1)
+    annotation (Line(points={{2,-10},{36,-10},{36,-22},{38,-22}},
+      color={255,0,255}));
+  connect(hys.y, logSwi1.u3)
+    annotation (Line(points={{2,-50},{36,-50},{36,-38},{38,-38}},
+      color={255,0,255}));
+  connect(logSwi1.u2, greThr.y)
+    annotation (Line(points={{38,-30},{30,-30},{30,-100},{2,-100}},
+      color={255,0,255}));
+  connect(div.u1, uQReq)
+    annotation (Line(points={{-82,26},{-100,26},{-100,60},{-140,60}},
+      color={0,0,127}));
+  connect(div1.u1, uQReq)
+    annotation (Line(points={{-82,86},{-100,86},{-100,60},{-140,60}},
+      color={0,0,127}));
+
+annotation (
+  defaultComponentName = "effCon",
+  Icon(
+    coordinateSystem(extent={{-100,-100},{100,100}}),
+    graphics={Rectangle(
+                extent={{-100,-100},{100,100}},
+                lineColor={0,0,127},
+                fillColor={255,255,255},
+                fillPattern=FillPattern.Solid),
+              Text(
+                extent={{-120,146},{100,108}},
+                lineColor={0,0,255},
+                textString="%name")}),
+  Diagram(
+    coordinateSystem(
+      preserveAspectRatio=false,
+      extent={{-120,-160},{120,120}})),
+  Documentation(
+    info="<html>
+    <p>
+    Efficiency condition used in staging up and down for boiler plants with both
+    condensing and non-condensing boilers. Implemented according to the
+    specification provided in 5.3.3.10, 1711 March 2020 Draft.
```

Comment:
Can you please specify the section number to the exact spots?

Reply:
I have listed the exact sequence points as subsections. Please let me know if that is the right way to indicate them.